HypothesisPlugin.inc.php

```diff
@@ -76,6 +84,83 @@ function changePdfjsPath($output, $templateMgr) {
 		return $newOutput;
 	}
 
+	/**
+	 * Adds Hypothesis tab configuration so sidebar opens automatically when PDF has annotations
+	 * @param $output string
+	 * @param $templateMgr TemplateManager
+	 * @return $string
+	 */
+	public function addHypothesisConfig($output, $templateMgr) {
+		if (preg_match('/<div[^>]+id="pdfCanvasContainer/', $output, $matches, PREG_OFFSET_CAPTURE)) {
+            $posMatch = $matches[0][1];
+			$config = $templateMgr->fetch($this->getTemplateResource('hypothesisConfig.tpl'));
+
+            $output = substr_replace($output, $config, $posMatch, 0);
+            $templateMgr->unregisterFilter('output', array($this, 'addHypothesisConfig'));
+        }
+		return $output;
+	}
+
+	public function addAnnotationNumberViewers($hookName, $args) {
+		$templateMgr = $args[0];
+		$template = $args[1];
+		$pagesToInsert = [
+			'frontend/pages/indexServer.tpl',
+			'frontend/pages/preprint.tpl',
+			'frontend/pages/preprints.tpl',
+			'frontend/pages/sections.tpl',
+			'frontend/pages/indexJournal.tpl',
+			'frontend/pages/article.tpl',
+			'frontend/pages/issue.tpl'
+		];
+
+		if (in_array($template, $pagesToInsert)) {
+            $request = Application::get()->getRequest();
```

Comment:
Here it seems that we have a different Indentation (spaces) than in other places (tabs) which is the general approach for v3.3. 

This is true also elsewhere in the plugin's code